src/OrchardCore.Modules/OrchardCore.Forms/Drivers/TextAreaPartDisplayDriver.cs

```diff
@@ -19,17 +21,21 @@ public override IDisplayResult Edit(TextAreaPart part, BuildPartEditorContext co
         {
             m.Placeholder = part.Placeholder;
             m.DefaultValue = part.DefaultValue;
+            m.Rows = part.Rows;
         });
     }
 
     public override async Task<IDisplayResult> UpdateAsync(TextAreaPart part, UpdatePartEditorContext context)
     {
-        var viewModel = new InputPartEditViewModel();
+        var viewModel = new TextAreaPartEditViewModel();
 
         await context.Updater.TryUpdateModelAsync(viewModel, Prefix);
 
         part.Placeholder = viewModel.Placeholder?.Trim();
         part.DefaultValue = viewModel.DefaultValue?.Trim();
+        part.Rows = viewModel.Rows > DefaultRows
```

Comment:
You should also make sure the user provide a value that is >= 1 but don't use default value. 


```suggestion
        part.Rows = viewModel.Rows;
```